README.md

```diff
@@ -35,7 +35,41 @@ If you do not specify this item in `composer.json` this utility will install to
 # Enabling
 
 ## Enabling PHP
-* `include_once __DIR__.'/base-cache-buster.php';` to a variable and then echo that when you reference your scripts, CSS, favicon etc.
+
+### Cache Buster
+
+In function.php
+
+    require_once __DIR__.'/packages/web-front-end-utilities/vendor/autoload.php';
+
+The cacheBuster will check the environment for the variable BASE_CACHE_BUSTER, unless WP_ENV is defined and set to 'development' in which case it will use a timestamp. This leaves how the version is set when you deploy open to whatever deployment tool you are using, such as Trellis or Capistrano.
+
+        $cacheBuster = new Base\CacheBuster\CacheBuster();
+
+        try{
+            $context['cachebuster'] = $cacheBuster->getVersion();
+        } catch (VersionUnidentifiableException $e) {
+            $context['cachebuster'] = '<fallback>'
```

Comment:
You can do both by encapsulating the creation of a `CacheBuster` object.

> without having an understanding of full OOP

I'm all for making APIs _simple_ to use, but don't think this should go against the design. You can create a wrapper around a complex API but that shouldn't impact the object itself. I'd argue for spending time teaching people OOP, rather than spending time making an API look easier (slightly false dichotomy).

Mocked up an example here, just an an exercise: https://gist.github.com/dave1010/d4bba9cc98eded14c8e0